packages/components/src/tooltip/index.tsx

```diff
@@ -64,17 +87,30 @@ function Tooltip( props: TooltipProps ) {
 	}
 	computedPlacement = computedPlacement || 'bottom';
 
+	// Disable reason: the hook can not run conditionally.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
 	const tooltipStore = Ariakit.useTooltipStore( {
 		placement: computedPlacement,
 		showTimeout: delay,
 	} );
 
+	if ( isNestedInParentTooltip ) {
```

Comment:
Did we also mean to trigger a console warning in that scenario (as per your [earlier suggestion](https://github.com/WordPress/gutenberg/pull/56490#issuecomment-1826427624))?